geotrek/signage/migrations/0030_auto_20230509_0816.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.2.18 on 2023-05-09 08:16
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def delete_blades_on_deleted_signages(apps, schema):
+    signage_model = apps.get_model('signage', 'Signage')
+    blade_model = apps.get_model('signage', 'Blade')
+    signages_deleted = signage_model.objects.filter(deleted=True)
+    for signage in signages_deleted:
+        for blade in blade_model.objects.filter(signage=signage):
+            blade.deleted = True
+            blade.save()
```

Comment:
```suggestion
            blade.save()
```
blade_model.objects.filter(signage__in=signage_deleted).update(deleted=True)

Reply:
You missed a 's' -> `signages_deleted`.
`blade_model.objects.filter(signage__in=signages_deleted).update(deleted=True)`
(to remove both for loops, nice one!)